### PR TITLE
enable user to access body of request

### PR DESCRIPTION
**Pull request by eirinnm**
Unless I'm mistaken, there doesn't appear to be a way to retrieve the content of an incoming request. This PR exposes the `raw_request` variable in the HTTPRequest object. I added a very basic usage example.

### CODE REVIEW

File: adafruit_httpserver.py [hunk 1]
```diff
@@ -287,7 +288,10 @@ def route(self, path: str, method: str = "GET"):
 
             @server.route(path, method)
             def route_func(request):
+                raw_text = request.raw_request.decode("utf8")
+                print("Received a request of length", len(raw_test), "bytes")
```
Comment (FoamyGuy) [inline adafruit_httpserver.py:292]: the variable created is `raw_text` but the one it tries to print is `raw_test` they'll need to match, text I think is more descriptive so:

```py
print("Received a request of length", len(raw_test), "bytes")
```

**Review by FoamyGuy — CHANGES_REQUESTED** [review]
Thanks for working on this @eirinnm! 

One small typo I noticed to change, but looks good to me beyond that.

File: adafruit_httpserver.py (hunk 1 above)
Comment (eirinnm) [reply adafruit_httpserver.py]: thanks for spotting it!

**Comment by tekktrik** [comment]
Looks like the typo was fixed, thanks @eirinnm  and @FoamyGuy!

**Pull request merged**